src/index.html

```diff
@@ -5,8 +5,413 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <!-- linthtml-configure attr-req-value="false" -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet"
+    crossorigin
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <a class= "container" href="#">
+        <img
+        src="https://raw.githubusercontent.com/mmahfuzi/layout_moyo-header-en/master/src/images/logo.png"
+        alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#laptops & computers"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <div class="cards-container">
+      <div class="cards-container__page page">
+        <div class="card-block">
+          <main data-qa="card" class="card">
+            <img
+              class="card__image image"
+              src="./images/imac.jpeg"
+              alt="Laptop"
+            >
+
+            <div class="card__text">
+              <p class="card__text card__text--1">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+              <p class="card__text card__text--2">Product code: 195434</p>
+            </div>
+
+            <div class="card__rate">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <div class="card__rate-review">Reviews: 5</div>
+            </div>
+
+            <div class="card__price">
+              <div class="card__price-text">
+                Price:
+              </div>
+
+              <div class="card__price-num">$2,199</div>
+            </div>
```

Comment:
```suggestion
            <p class="card__price">
              <span class="card__price-text">
                Price:
              </span>
              <span class="card__price-num">
                 $2,199
               </span>
            </p>
```
Not a mistake, but since it's basically a text split in two, you can wrap it in a p tag and use span for each text. This way you can tell what's the content of this block and for semantic purposes.